samples/snippets/csharp/VS_Snippets_CLR/Exception.Throwing/CS/throw.cs

```diff
@@ -1,32 +1,35 @@
-﻿// <Snippet1>
-using System;
+﻿using System;
 using System.IO;
 
-public class ProcessFile
+namespace Example
 {
-    public static void Main()
+    public class ProcessFile
     {
-        FileStream fs;
-        try
+        public static void Main()
         {
-            // Opens a text tile.
-            fs = new FileStream(@"C:\temp\data.txt", FileMode.Open);
-            var sr = new StreamReader(fs);
+            // <Snippet1>
+            FileStream? fs = null;
```

Comment:
Being a fan of `var`, I'd consider this alternative:

```suggestion
            var fs = default(FileStream);
```